docs/other/php-7.md

```diff
@@ -23,11 +23,21 @@ php_packages:
   - php7.0-mbstring
 php_mysql_package: php7.0-mysql
 php_fpm_daemon: php7.0-fpm
+php_conf_paths: 
+  - /etc/php/7.0/fpm
+  - /etc/php/7.0/apache2
+  - /etc/php/7.0/cli
+php_extension_conf_paths:
+  - /etc/php/7.0/fpm/conf.d
+  - /etc/php/7.0/apache2/conf.d
+  - /etc/php/7.0/cli/conf.d
 php_fpm_pool_conf_path: "/etc/php/7.0/fpm/pool.d/www.conf"
 ```
 
 Also, comment out `xhprof`, `xdebug`, `redis` and `memcached` from the `installed_extras` list, as these extensions are not yet supported for PHP 7 (as of late 2015).
 
+Xdebug 2.4.0 works with PHP 7.0. Sources available at https://xdebug.org/files/xdebug-2.4.0.tgz but you'll need to update `ansible-role-php-xdebug` to pick the source from there.
```

Comment:
I've upped the default in that role to 2.4.0, which should work fine with PHP 5.4-7.x, so no need for this extra line anymore. Can you remove it?